release_notes/osd_latest_product_updates.adoc

```diff
@@ -35,437 +35,277 @@ page for their scheduled upgrade date. This topic has been updated to reflect
 the upcoming new features and changes.
 ====
 
-The latest update of Red Hat {product-title} uses  link:https://docs.openshift.com/container-platform/3.6/release_notes/ocp_3_6_release_notes.html[Red Hat OpenShift Container Platform version 3.6], which is based on
-link:https://github.com/openshift/origin/releases/tag/v3.6.0[OpenShift Origin 3.6]. New features, changes, bug fixes, and known issues that
+The latest update of Red Hat {product-title} uses  link:https://docs.openshift.com/container-platform/3.7/release_notes/ocp_3_7_release_notes.html[Red Hat OpenShift Container Platform version 3.7], which is based on
+link:https://github.com/openshift/origin/releases/tag/v3.7.0[OpenShift Origin 3.7]. New features, changes, bug fixes, and known issues that
 pertain to the latest updates of {product-title} are included in this topic.
 
 [[osd-new-features-and-enhancements]]
 == New Features and Enhancements
 
+[[osd-dedicated-admin-updates]]
+=== Dedicated Administrator Updates
+
+[[osd-service-accounts]]
+==== Service Accounts
+As of {product-title} on 3.7, a reserved project called 'dedicated-admin' will be created and maintained on each cluster. Any service account created within this project will have dedicated-admin permissions by default. Only dedicated-admin users will be able to manage service accounts within this project.
```

Comment:
Consider marking the 5 instances of the "dedicated-admin" project/user name as

```
*dedicated-admin*
```

?